templates/parts/show.html.twig

```diff
@@ -0,0 +1,50 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+<section class="nameMotor">
+    <p class="box"> Moto de Baptiste <br> KTM 125 sx </p>
+</section>
+
+<div class="flex parts">
+    <h2>Nombre d'heure</h2>
+
+    <div class="namePart">
+        <label for="partOne">
+            <p>nom de pièce</p>
+            <input type="checkbox" name="my-checkbox" id="partOne">
+        </label>
+    </div>
+        <progress value = "70" max = "100"> </progress>
+
+    <div class="namePart">
+        <label for="partTwo">
+            <p>nom de pièce</p>
+            <input type="checkbox" name="my-checkbox" id="partTwo">
+        </label>
+    </div>
```

Comment:
if the input is in the label, you don't need for/id... and you don't div and p here ;)